### PR TITLE
Various bug fixes in prescriptions

### PR DESCRIPTION
**Pull request by saneshark**

### CODE REVIEW

File: app/serializers/tracking_serializer.rb [hunk 1]
```diff
@@ -5,6 +5,13 @@ def id
 
   link(:self) { rx_v1_prescription_trackings_url(object.prescription_id) }
   link(:prescription) { rx_v1_prescription_url(object.prescription_id) }
+  link(:tracking_url) do
+    if object.delivery_service.casecmp("UPS")
+      "https://wwwapps.ups.com/WebTracking/track?track=yes&trackNums=#{object.tracking_number}"
+    else
```
Comment (ayaleloehr) [inline app/serializers/tracking_serializer.rb]: It should only be UPS and USPS, but I'm worried about funky edge cases or wrong data being returned to us somehow. Can we have an explicit `else if` for USPS and then the final `else` that catches everything else be like a blank link or something?

**Comment by ayaleloehr** [comment]
LGTM with one nit to likely fix about tracking carrier edge cases.

**Comment by ayaleloehr** [comment]
@saneshark with this shared repo, I get nervous about having our schemas directly in `/spec/support/schemas` instead of something like `/spec/support/schemas/rx` that separates by project. Is that possible/easy to do?  I just think /schemas is going to get crowded if we don't.  (similar comment about `/spec/serializers` as well).

**Comment by saneshark** [comment]
@ayaleloehr that is definitely something that can be done, but can we revisit that as it becomes a problem or open a separate issue to address it?

In the case of the serializers, it's slightly annoying to not have them all in one folder because of how ActiveModelSerializer looks them up based on the controller class resource. For schemas it's trivial to change, but those will mostly likely get moved to the common gem or submodule that Mark Olson created.

**Comment by ayaleloehr** [comment]
This makes sense. I can get behind that. 

LGTM. will be interesting to see if others across the projects have opinions on factorygirl being added, but going to merge and we can bikeshed that later as desired.

**Pull request merged**